### PR TITLE
Make sure randomSort does not mutate store

### LINKED ISSUE #582: Sorting randomly should copy array first
##### Reproduction steps

1. Go to home
2. Go to artists
3. See that artists are sorted randomly

##### Expected behaviour

Artists should not be sorted randomly in artists view

### PR DESCRIPTION
**Pull request by robbevp**
Fixes #582

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -218,11 +218,12 @@ export default {
       return items;
     },
     randomSort(items) {
-      for (let i = items.length - 1; i > 0; i--) {
+      const newItems = items;
```
Comment (chvp) [inline src/views/Home.vue:221]: This is not a copy. It's a copy of the reference, so any mutations here will also show up in the original array.

**Review by chvp — CHANGES_REQUESTED** [review]

File: src/views/Home.vue (hunk 1 above)
Comment (robbevp) [reply src/views/Home.vue:221]: Weird, since it does work. If I look in the Vue devtools, `this.albums` and `this.randomAlbums` are ordered differently

Comment (chvp) [reply src/views/Home.vue:221]: Maybe it's a dev mode vs prod mode issue?

Comment (chvp) [reply src/views/Home.vue:221]:
```
> const a = [1, 2]
undefined
> const b = a;
undefined
> [b[0], b[1]] = [b[1], b[0]]
[ 2, 1 ]
> a
[ 2, 1 ]
```
This is essentially what this code does, as you can see, it doesn't work.

Comment (robbevp) [reply src/views/Home.vue]: I could see the issue before in dev mode. Anyway, I changed the code

**Review by chvp — APPROVED** [review]

**Pull request merged**